programs/mango-v4/src/instructions/serum3_place_order.rs

```diff
@@ -204,7 +204,7 @@ pub fn serum3_place_order(
     ctx: Context<Serum3PlaceOrder>,
     side: Serum3Side,
```

Comment:
I have not added a reduce_only order type for serum, only market can have reduce only, wdyt?